scripts/globusmonitor/globus_uploader_service.py

```diff
@@ -67,18 +61,6 @@ def safeCopy(obj):
 
     return newObj
 
-"""If metadata keys have periods in them, Clowder will reject the metadata"""
```

Comment:
Why was this removed?

Reply:
our terrautils md cleaning code should perform this, it's code that should be done anywhere we read json metadata from file (e.g. cleaner extractor) and right now it's just here. hasn't been an issue since it only applies IIRC to early 2016 md files.

https://github.com/terraref/terrautils/pull/48 Here's what I'm thinking.